sci-libs/metis/metis-5.1.0-r1.ebuild

```diff
@@ -55,16 +55,16 @@ src_test() {
 
 src_install() {
 	cat >> "${T}"/metis.pc <<- EOF
-	prefix="${EPREFIX}"/usr
-	exec_prefix="${EPREFIX}"/usr
-	libdir="${EPREFIX}"/usr/$(get_libdir)
-	includedir="${EPREFIX}"/usr/include
+	prefix=${EPREFIX}/usr
+	exec_prefix=\${prefix}
+	libdir=\${prefix}/$(get_libdir)
```

Comment:
While in this case it doesn't make a big difference, pkg-config files have a canonical layout, see
https://people.freedesktop.org/~dbn/pkg-config-guide.html

i.e. change `libdir=\${prefix}/$(get_libdir)` to `libdir=\${exec_prefix}/$(get_libdir)`

And maybe swap `libdir` and `includedir` around.